### PR TITLE
Color: cleanup and refactor status related aliases

### PR DESCRIPTION
**Pull request by ianmcburnie**
So, there will likely be a flurry of activity now as I cleanup as much as I can just before pre-release :-)

While going through the token normalization and dark mode, I noticed a few things that could be cleaned up regarding colours.

One of the mains things was that `color-attention-border-color`, `color-attention-background-color` and `color-attention-icon-color` could just be simplified & consolidated down to one alias, I chose: `color-status-attention` (although design systems prefers the term, "alerts"). Same goes for confirmation and information. I'm trying to make the number of product aliases as concise as possible.

I also deleted a bunch of ancient ds4 aliases, and reshuffled a few other things. 

Regarding playbook and figma, it seems a little bit of a moving target at the moment, so I haven't tried to cover everything (for example highlight, disabled text alt, default text alt). We can take another shot at it when it's finalized on their end.

Figma
Screen Shot 2020-09-09 at 7 56 28 PM

Playbook
Screen Shot 2020-09-09 at 7 36 59 PM

Skin before
Screen Shot 2020-09-09 at 7 36 18 PM

Skin after

Screen Shot 2020-09-11 at 11 57 15 AM

### CODE REVIEW

File: docs/src/less/base/index.less [hunk 1]
```diff
@@ -233,7 +233,7 @@ span.demo__inner {
 /* CAROUSEL SECTION */
 
 .carousel__list--default-demo li {
-    background-color: @color-information-background;
+    background-color: @color-status-information;
```
Comment (agliga) [inline docs/src/less/base/index.less:236]: This color looks off on the carousel and doesn't have a good color contrast on it. 
Screen Shot 2020-09-11 at 11 43 09 AM

Comment (ianmcburnie) [reply docs/src/less/base/index.less:236]: I think it's currently like this in production, right? If so, we should fix, but in a different PR.

Comment (ianmcburnie) [reply docs/src/less/base/index.less:236]: Oh scrap that - I see i used the wrong variable. Will fix.

Comment (ianmcburnie) [reply docs/src/less/base/index.less]: Fixed. Good spot!

Screen Shot 2020-09-11 at 11 54 13 AM

**Review by agliga — APPROVED** [review]

**Pull request merged**